prqlc/prql-compiler/tests/sql/sql.rs

```diff
@@ -3329,7 +3363,10 @@ fn test_exclude_columns() {
       tracks
     "###
     );
+}
 
+#[test]
+fn test_exclude_columns_07() {
```

Comment:
Interesting — do you prefer them like this?

One advantage of doing it this way is seeing exactly what fails, particularly in non-insta tests which don't `FORCE_PASS`...

Reply:
That was because one of the tests was panicking and it wasn't saying which one.

Reply:
Yeah, I aesthetically prefer them as one big test, but then sometimes I find it doesn't give quite enough info...